Studies/source/GenStudy.cxx

```diff
@@ -529,15 +481,46 @@ class GenStudy {
                     }
                 }
             }
+            if(b_jet_matches_total.size() == 2 && tau_matches_total.size() ==2) {
```

Comment:
here it should control for double-counting to exclude cases when, for example, one tau has 2 matches, while other has 0 (same for b). Or it was done before?